src/modules/Channel/components/MessageInputWrapper/index.tsx

```diff
@@ -15,18 +16,33 @@ export interface MessageInputWrapperProps {
 }
 
 export const MessageInputWrapper = (props: MessageInputWrapperProps) => {
+  const { config } = useSendbirdStateContext();
   const context = useChannelContext();
   const {
     currentGroupChannel,
+    localMessages,
     sendMessage,
     sendFileMessage,
     sendVoiceMessage,
     sendMultipleFilesMessage,
   } = context;
 
+  const lastMessage = currentGroupChannel?.lastMessage;
+  const isLastMessageSuggestedRepliesEnabled = config?.groupChannel?.enableSuggestedReplies
+    && lastMessage
+    && getSuggestedReplies(lastMessage).length > 0
+    && (
+      !localMessages
+      || localMessages.length === 0
+      || localMessages.every((message) => (message as UserMessage).sendingStatus === 'succeeded')
+    );
```

Comment:
In my opinion, using localMessages?.length === 0 seems sufficient. What do you think?